subxt/src/client.rs

```diff
@@ -83,6 +85,14 @@ impl ClientBuilder {
         self
     }
 
+    /// Set the metadata.
+    ///
+    /// *Note:* Metadata will no longer be downloaded from the runtime node.
+    pub fn set_metadata(mut self, metadata: Metadata) -> Self {
```

Comment:
```suggestion
    #[cfg(test)]
    pub fn set_metadata(mut self, metadata: Metadata) -> Self {
```

Reply:
Unfortunately, `cfg(test)` is not active for integration tests. I would place this under `integration_tests` feature and follow up with #515  😄 .